ChemistryLib/PhreeqcKernelData/CreateInitialAqueousSolution.cpp

```diff
@@ -62,6 +62,17 @@ InitialAqueousSolution createInitialAqueousSolution(
             "for the transport process of hydrogen.");
     }
 
+    auto const charge_balance =
+        //! \ogs_file_param{prj__chemical_system__solution__charge_balance}
+        config.getConfigParameter<std::string>("charge_balance", "");
+
+    if (charge_balance == "pH")
```

Comment:
I recommend to check for other cases too. If empty, then OK, but if not empty it could be mistyped or from future ogs or somethings else, then OGS_FATAL.

Reply:
Creation function of charge balance is moved into the folder `Common`, shared with the both of Phreeqc Interface.